.github/workflows/unittests-gpu.yml

```diff
@@ -57,12 +57,12 @@ jobs:
           echo "Start submitting job"
           python ./tools/batch/submit-job.py --region us-east-1 \
                                              --job-type g4dn.4x \
-                                             --name GluonNLP-${{ github.ref }}-${{ github.run_id }} \
+                                             --name GluonNLP-GPU-Test-PR#${{ github.event.number }} \
                                              --source-ref ${{ github.event.pull_request.head.ref }} \
                                              --work-dir tools/batch \
                                              --save-path temp \
                                              --remote https://github.com/${{ github.event.pull_request.head.repo.full_name }} \
-                                             --command "./batch_states/test.sh | tee > ./script.log" \
+                                             --command "(./batch_states/test.sh | tee > ./gputest.stdout.log) 3>&1 1>&2 2>&3 | tee ./gputest.stderr.log" \
```

Comment:
You need to retrieve the log from cloudwatch. Just looking at the stdout and stderr of this command is not sufficient